python-frontend/src/test/java/org/sonar/python/parser/compound/statements/CasePatternsTest.java

```diff
@@ -81,4 +81,29 @@ public void capture() {
     assertThat(p)
       .matches("x");
   }
+
+  @Test
+  public void sequence() {
+    assertThat(p)
+      .matches("[]")
+      .matches("['foo', 'bar']")
+      .matches("['foo', x]")
+      .matches("['foo', *tail]")
+      .matches("[head, *tail]")
+      .matches("['foo' as head]");
+
+    assertThat(p)
+      .matches("()")
+      .matches("(x,)")
+      .matches("('foo', 'bar')")
+      .matches("(head, *tail)");
+
+    setRootRule(PythonGrammar.SEQUENCE_PATTERN);
+    assertThat(p).notMatches("(x)"); // this is a group pattern instead
+
+    setRootRule(PythonGrammar.CASE_BLOCK);
+    assertThat(p)
+      .matches("case 'foo', x, z: ...");
+
+  }
```

Comment:
I'm not sure it's a good idea to add tests here because we don't test that `PythonTreeMaker` against those pieces of code. We could add those cases to `PythonTreeMakerMatchStatementTest` to verify in a very concise way that:
* `PythonTreeMaker` doesn't crash on them
* The `Tree` instance has all tokens.

Reply:
That's a fair point, I will move them to `PythonTreeMakerMatchStatementTest ` class